src/prototype_room_my.js

```diff
@@ -540,29 +540,24 @@ Room.prototype.reviveRoom = function() {
 
       let distance = Game.map.getRoomLinearDistance(this.name, roomName);
       if (distance < config.nextRoom.maxDistance) {
-        let creepToSpawn = {
-          role: 'nextroomer',
-          routing: {
-            targetRoom: this.name
-          }
-        };
+        let route = this.findRoute(roomOther.name, this.name);
+        // TODO Instead of skipping we could try to free up the way: nextroomerattack or squad
+        if (route.length === 0) {
+          roomOther.log('No route to other room: ' + roomOther.name);
+          continue;
```

Comment:
ok, got it. it should be used so i supose in canHelp too i supose as i did a check for dist < cst. you can change dist value using getRoute